### ISSUE TITLE
Ideas for the far future of Taiga

### ISSUE THREAD

**Issue by Zkaface** [TeamFRM/TAIGA#108]
Please comment ideas, what you like to see in Taiga!

Here my suggestions for future releases:

- two special dimensions with high end ores
- each dimension with strong and even stronger mobs
- maybe additional mechanisms of the smeltery (additional blocks adding features)

**Comment by Flabort** [comment]
Yes, in the vein of the third checkbox, I was thinking, perhaps a block that can replace the air blocks inside a smeltery; it transfers RF, and requires it, so you need to make the pillar of it as tall as the smeltery so you can connect a wire to the top of it. It's model is that of some rotating blades like in a blender. It uses the RF to increase melting speed by 0.5% of the current speed, and to mix certain alloys that don't automatically mix in the smeltery - EG it forces metals to alloy like mixing vinegar and oil into salad dressing. Not all alloys would require it, only certain ones.

Love the dimension ideas. But why not make it 4 dimensions, to reflect the 4x categories of ore ( Arcane <> Solide <> Ethere <> Ratio <> Arcane), unless you've given up on that. Solide could be Abyss, Hellfire Depth could be Ratio, and then for Ethere there could be Asgard, and for Arcane there could be Wonderland.
